cockatrice/src/player.cpp

```diff
@@ -292,21 +310,31 @@ Player::Player(const ServerInfo_User &info, int _id, bool _local, bool _judge, T
         libraryMenu->addAction(aAlwaysRevealTopCard);
         libraryMenu->addAction(aAlwaysLookAtTopCard);
         libraryMenu->addSeparator();
-        libraryMenu->addAction(aMoveTopToPlay);
-        libraryMenu->addAction(aMoveTopToPlayFaceDown);
-        libraryMenu->addAction(aMoveTopCardToBottom);
-        libraryMenu->addAction(aMoveBottomCardToGrave);
-        libraryMenu->addSeparator();
-        libraryMenu->addAction(aMoveTopCardToGraveyard);
-        libraryMenu->addAction(aMoveTopCardToExile);
-        libraryMenu->addAction(aMoveTopCardsToGraveyard);
-        libraryMenu->addAction(aMoveTopCardsToExile);
+        topLibraryMenu = libraryMenu->addTearOffMenu(QString());
+        bottomLibraryMenu = libraryMenu->addTearOffMenu(QString());
         libraryMenu->addSeparator();
         libraryMenu->addAction(aOpenDeckInDeckEditor);
         deck->setMenu(libraryMenu, aDrawCard);
-    } else {
-        handMenu = nullptr;
-        libraryMenu = nullptr;
+
+        topLibraryMenu->addAction(aMoveTopToPlay);
+        topLibraryMenu->addAction(aMoveTopToPlayFaceDown);
+        topLibraryMenu->addAction(aMoveTopCardToBottom);
+        topLibraryMenu->addSeparator();
+        topLibraryMenu->addAction(aMoveTopCardToGraveyard);
+        topLibraryMenu->addAction(aMoveTopCardToExile);
+        topLibraryMenu->addAction(aMoveTopCardsToGraveyard);
+        topLibraryMenu->addAction(aMoveTopCardsToExile);
+
+        bottomLibraryMenu->addAction(aDrawBottomCard);
+        bottomLibraryMenu->addAction(aDrawBottomCards);
+        bottomLibraryMenu->addAction(aMoveBottomToPlay);
```

Comment:
Seperate `Draw` (put in your hand) the same as we seperate `Play` (put to stack/battlefield) from `Move` (put to gy/ex)
```suggestion
        bottomLibraryMenu->addAction(aDrawBottomCards);
        bottomLibraryMenu->addSeparator();
        bottomLibraryMenu->addAction(aMoveBottomToPlay);
```